static/app/utils/useResizableDrawer.mdx

```diff
@@ -0,0 +1,166 @@
+import {useState} from 'react';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {Flex} from 'sentry/components/container/flex';
+import {Button} from 'sentry/components/core/button';
+
+import {useResizableDrawer} from './useResizableDrawer';
+
+## useResizableDrawer
+
+Use this hook to create a resizable drawer. The hook will return the props to pass to the element you wish to resize and the props to pass to the resize handle.
+
+The hook returns the following:
+
+- `resizing`: A boolean that indicates if the drawer is resizing.
+- `resizeHandleProps`: The props to pass to the resize handle.
+- `resizedElementProps`: The props to pass to the element you wish to resize.
+
+<CodeSnippet language="typescript" dark>
+```tsx
```

Comment:
not part of this PR but I wish it knew to render a codesnippet based on the 3 ticks and the language marker

Reply:
I thought there was some easy way to tell mdx what components to use when rendering things like codeblocks?